README.md

```diff
@@ -212,6 +215,63 @@ metadata:
 [...]
 ```
 
+### Pager teams
+
+Instead of specifying the authentication token directly in the check or agent annotations, you can instead reference a pager team name, which will then be used to lookup the corresponding token from the handler environment. 
+Corresponding pager team token environment variables can be populated in the handler environment in 3 different ways
+1. Explicitly set in the handler definition
+2. Kept as Sensu [secrets][13] and referenced in the handler definition
+3. Defined in the [backend service environment file][14], read in at backend service start.
+
+Pager team names will be automatically suffixed with configured --pagerduty-team-suffix (default: `_pagerduty_suffix`)
```

Comment:
--pagerduty-team-suffix doesn't square with the app's flags

Reply:
fixed in latest commit